rules/CodingStyle/Rector/Class_/AddArrayDefaultToArrayPropertyRector.php

```diff
@@ -177,11 +180,13 @@ private function clearNotNullBeforeCount(Class_ $class, array $propertyNames): v
                     return null;
                 }
 
-                if (! isset($node->args[0])) {
```

Comment:
Would /`getArgs` help with all thes?

Reply:
possibly, I will create separate PR for `getArgs()` usage.

Reply:
That's seems not possible as my comment at PR https://github.com/rectorphp/rector-src/pull/911#issuecomment-924981030